### PR TITLE
[Modular][Ready] Few bits of clothing

### PR DESCRIPTION
**Pull request by ArthurStan**
## About The Pull Request
My first PR to Skyrat, doing this mostly as a test run to see if i still got the handle of things. Trying to port some clothes from Paradise with a few tweaks of my own.
image
image

Feedback is more than appreciated, let me know if i forgot something, or messed up somehow.

## How This Contributes To The Skyrat Roleplay Experience
One of the clothing pieces is a coat which i recoulored and renamed to mention an faction from the lore, seeing as SR's wiki shares the same lore as Paradise, i figure that having that could add some possibilities for future RPs, if it works, i plan on adding more gear from said faction, or other factions as well.

The other clothes are mostly basic stuff which i noticed were missing. Some suits 'n such.

## Changelog

:cl:
add: New clothes and a coat
/:cl:

### CODE REVIEW

**Comment by Zonespace27** [comment]
CL isn't formatted right, it has to be formatted like this:

`qol: thing here`
`add: the thing I added`

File: modular_skyrat/modules/customization/modules/clothing/under/misc.dm [hunk 1]
```diff
@@ -733,3 +733,36 @@
...
+	name = "working joe"
+	desc = "You're here to work, not to look fancy."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
+	icon_state = "joesuit"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+
+/obj/item/clothing/under/suit/redbureaucrat
+	name = "bureaucrat in red"
+	desc = "Climbing the ranks, no matter the consequences"
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/under/misc.dm]:
```suggestion
	desc = "Climbing the ranks, no matter the consequences."
```

**Review by tf-4 — CHANGES_REQUESTED** [review]
Put screenshot of the sprites in game in the PR body pls

**Comment by tf-4** [comment]
Needs the images of them in game if you can, rather than in the editor.
It's easier to get an idea of how they actually look, as well as helping us ensure at a glance that they're working properly and stuff. You'd be surprised what people don't test.

**Pull request closed without merging**